main.tf

```diff
@@ -25,11 +25,12 @@ resource "google_kms_key_ring" "key_ring" {
 }
 
 resource "google_kms_crypto_key" "key" {
-  count           = var.prevent_destroy ? length(var.keys) : 0
-  name            = var.keys[count.index]
-  key_ring        = google_kms_key_ring.key_ring.id
-  rotation_period = var.key_rotation_period
-  purpose         = var.purpose
+  count              = var.prevent_destroy ? length(var.keys) : 0
+  name               = var.keys[count.index]
+  key_ring           = google_kms_key_ring.key_ring.id
+  rotation_period    = var.key_rotation_period
+  purpose            = var.purpose
+  crypto_key_backend = var.crypto_key_backend
```

Comment:
@romanini-ciandt I think we will need to bump the min version since this was added recently 

https://github.com/hashicorp/terraform-provider-google/blob/main/CHANGELOG.md#5230-apr-1-2024

Reply:
Good catch @bharathkkb - Quick fix: #139